subjects/devops/string_tokenizer_count/README.md

```diff
@@ -0,0 +1,42 @@
+## string_tokenizer_count
+
+### Instructions
+
+Create a file string_tokenizer_count.py that contains a function tokenizer_counter which takes in a string as a parameter and returns a dictionary of words and their count in the string.
```

Comment:
I would add some formatting to better highlight the name of the expected file and expected function. For example

`string_tokenizer_count.py` and `tokenizer_counter`

Reply:
Thanks. Done.